docs/reference/ccr/disaster-recovery.asciidoc

```diff
@@ -0,0 +1,140 @@
+[role="xpack"]
+[[ccr-disaster-recovery-tutorial]]
+=== Tutorial: Configure disaster recovery with {ccr}
+++++
+<titleabbrev>Configure disaster recovery</titleabbrev>
+++++
+
+////
+[source,console]
+----
+PUT /server-metrics
```

Comment:
I don't understand why we're setting up this index if it's not referenced at all in the document. Should `my_index` in the later examples instead use `server-metrics`?